helm lint fails with helm 3: chart type is not valid in apiVersion 'v1'. It is valid in apiVersion 'v2'

Steps to recreate:
```
➜ git clone git@github.com:zloeber/helm-namespace.git
➜ cd helm-namespace/chart/namespace
➜ helm dependency build .
➜ helm lint
==> Linting .
[INFO] Chart.yaml: icon is recommended
[ERROR] Chart.yaml: chart type is not valid in apiVersion 'v1'. It is valid in apiVersion 'v2'

Error: 1 chart(s) linted, 1 chart(s) failed
➜  helm version
version.BuildInfo{Version:"v3.0.2", GitCommit:"19e47ee3283ae98139d98460de796c1be1e3975f", GitTreeState:"clean", GoVersion:"go1.13.5"}
```

--- a/chart/namespace/Chart.yaml
+++ b/chart/namespace/Chart.yaml
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
 name: namespace
 description: A generic helm3 namespace chart
 type: application

